src/fluree/server/consensus/standalone.clj

```diff
@@ -0,0 +1,135 @@
+(ns fluree.server.consensus.standalone
+  (:require [clojure.core.async :as async :refer [<! >! go]]
+            [fluree.db.api :as fluree]
+            [fluree.db.constants :as const]
+            [fluree.db.util.async :refer [<? go-try]]
+            [fluree.db.util.core :refer [get-first-value]]
+            [fluree.db.util.log :as log]
+            [fluree.server.consensus :as consensus]
+            [fluree.server.consensus.msg-format :as msg-format]
+            [fluree.server.handlers.shared :refer [deref!]]
+            [fluree.server.subscriptions :as subscriptions]
+            [fluree.server.watcher :as watcher]))
+
+(defrecord StandaloneTransactor [tx-queue]
+  consensus/TxGroup
+  (queue-new-ledger [_ ledger-id tx-id txn opts]
+    (go
+      (let [event-msg (msg-format/queue-new-ledger ledger-id tx-id txn opts)]
+        (>! tx-queue event-msg)
```

Comment:
At some point we need to use `put!` instead of `>!` and maybe you are thinking of that upstream, else we can never return a response and the whole server is locked.

If put! is false, the"requestor" should get a 503 server busy response or equivalent (but the server still might be able to respond to queries, for example). Hopefully it could catch up with backlog and accept new tx requests soon.

If we block all the way up the stack, nothing can happen.

Reply:
I don't think we can use `put!` because we'll run into a "too many pending puts" error under heavy load. 

We could wrap upstream code with a timeout channel to handle returning the 503s, or we could put a dropping buffer on the tx-queue channel and bubble up the return value of `>!` (as long as the return value is false when the buffer is full, but I'd have to verify that this is the case).

Reply:
I must have misunderstood `put!`. I thought it was exactly for this reason and would return false (and not try to `>!` if the buffer was full). I thought there was something like that, maybe a different fn name?

Reply:
Here it is: https://clojuredocs.org/clojure.core.async/offer!